src/applications/vaos/project-cheetah/redux/reducer.js

```diff
@@ -395,7 +396,10 @@ export default function projectCheetahReducer(state = initialState, action) {
     case FORM_REQUEST_CURRENT_LOCATION_FAILED: {
       return {
         ...state,
-        requestLocationStatus: FETCH_STATUS.failed,
+        newBooking: {
+          ...state.newBooking,
+          requestLocationStatus: FETCH_STATUS.failed,
```

Comment:
Because we were setting state in the wrong spot, a location errors was leaving a spinner up indefinitely.